scipy/signal/filter_design.py

```diff
@@ -4843,6 +4845,152 @@ def _design_notch_peak_filter(w0, Q, ftype, fs=2.0):
     return b, a
 
 
+def _hz_to_erb(hz):
+    """
+    Utility for converting from frequency (Hz) to the
+    Equivalent Rectangular Bandwith (ERB) scale
+    """
+    EarQ = 9.26449
+    minBW = 24.7
+    order = 1
+    return ((hz / EarQ) ** order + minBW ** order) ** (1 / order)
+
+
+def iirgammatone(freq, fs=16000, output='ba'):
+    """
+    IIR gammatone filter design.
+
+    This function computes the coefficients of an 8th order IIR
+    digital filter, which models a 4th order gammatone filter.
+
+    Parameters
+    ----------
+    freq : float
+        Center frequency of the filter in Hz.
```

Comment:
Drive-by comment: Don't specify explicit units (Hz) here.  This is the frequency, in whatever units are determined by `fs`.